srcpkgs/glibc/template

```diff
@@ -1,14 +1,16 @@
 # Template file for 'glibc'
 pkgname=glibc
-version=2.32
-revision=2
+version=2.36
+revision=1
+_patchver="72-g0f90d6204d"
+wrksrc="glibc-${version}-${_patchver}"
 bootstrap=yes
 short_desc="GNU C library"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later, BSD-3-Clause"
 homepage="http://www.gnu.org/software/libc"
-distfiles="${GNU_SITE}/glibc/glibc-${version}.tar.xz"
-checksum=1627ea54f5a1a8467032563393e0901077626dc66f37f10ee6363bb722222836
+distfiles="https://vasilek.cz/paste/glibc-${version}-${_patchver}.tar.xz"
```

Comment:
glibc has a release branch where patches go to including several critical ones.
https://github.com/bminor/glibc/tree/release/2.36/master
paper didn't want me to include it as a patch (like what debian does) since the patch is enormous.
So we generated a tarball from the release branch using make dist. (Fedora does this and Arch just uses git)
It was generated from the following commit:
https://github.com/bminor/glibc/commit/0f90d6204d79223fd32248c774df0cb7f0e604de